test/jb_token/mint.test.js

```diff
@@ -0,0 +1,60 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+
+describe('JBToken::mint(...)', function () {
+  const PROJECT_ID = 10;
+  const name = 'TestTokenDAO';
+  const symbol = 'TEST';
+
+  let deployer;
+  let addrs;
+  let testToken;
+
+  before(async function () {
+    [deployer, ...addrs] = await ethers.getSigners();
+    testToken = await deployToken(name, symbol);
+  });
+
+  async function deployToken(name, symbol) {
+    const jbTokenFactory = await ethers.getContractFactory('JBToken');
+    const jbToken = await jbTokenFactory.deploy(name, symbol);
+    await jbToken.deployed();
+    return jbToken;
+  }
```

Comment:
Let's use the `await setup()` pattern that I've introduced in the JBDirectory tests, as opposed to `before()`

Reply:
Before I do - any reasoning for this? The before() patterns seems to achieve the same with less lines of code

Reply:
The preference is to not introduce stateful variables across multiple tests in the suite.

Reply:
Gotcha, I can get behind that